internal/interpreter/api.rs

```diff
@@ -576,7 +576,11 @@ impl ComponentCompiler {
         file_loader_fallback: impl Fn(
                 &Path,
             ) -> core::pin::Pin<
-                Box<dyn core::future::Future<Output = Option<std::io::Result<String>>>>,
+                Box<
```

Comment:
This is a breaking change on public API

Reply:
Any idea on how to avoid it?

Keep this one as is, add a new callback function and use that if set and fall back to the current one?